### PR TITLE
Feature/admin update suggested meals

### PR DESCRIPTION
**Pull request by olasubomi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,13 +27,20 @@
     "material-ui-chip-input": "^2.0.0-beta.2",
     "mongodb": "^3.5.5",
     "mongoose": "^5.9.7",
+    "multer": "^1.4.2",
     "nodemailer": "^6.4.6",
     "passport": "^0.4.1",
     "passport-facebook": "^3.0.0",
     "passport-google-oauth": "^2.0.0",
     "passport-local": "^1.0.0",
+<<<<<<< Updated upstream
```
Comment (olasubomi) [inline package.json:36]: I can not run the latest commit due to this merge conflict

Comment (olasubomi) [reply package.json:36]: server_errors

Comment (olasubomi) [reply package.json:36]: client_display

File: server.js [hunk 2]
```diff
@@ -148,11 +147,14 @@ app.post("/api/forgotpass", forgotPassword);
...
 app.post("/api/signupuser", signupCustomer);
 app.post("/api/signup/:newcustomerId", authenticationSignup);
-app.post("/api/addMealSuggestion/", addMealSuggestion);
+app.post("/api/addMealSuggestion/", upload.single('imgSrc'), addMealSuggestion);
+app.post("/api/updateSuggestItem/", upload.single('imgSrc'), updateSuggestedMealItem);
 
 // app.post('/api/appendItem',appendItem)
 app.delete("/api/remove-list/:customerId", removeList);
 app.delete("/api/remove-item/:idItem/:customerId", removeItem);
+app.get("/api/removeSeggestItem/:suggestedMealID", removeSuggestedMealItem);
```
Comment (olasubomi) [inline server.js:156]: removeSuggestion not removeSeggestion

File: server.js [hunk 3]
```diff
@@ -350,3 +352,57 @@ app.listen(port, () => console.log(`Listening on port ${port}`));
 //             res.redirect('/');
 //         }
 //     });
+
```
Comment (olasubomi) [inline server.js:355]: Can we delete this code that youve created and commented out ?

File: package.json (hunk 1 above)
Comment (devradoje) [reply package.json:36]: Did you install node-module? If you didn't install it, please install and check it again.

**Pull request merged**